testsuite/cli/src/client_proxy.rs

```diff
@@ -568,12 +568,11 @@ impl ClientProxy {
         let (address, _) = self.get_account_address_from_parameter(space_delim_strings[1])?;
         let private_key = Ed25519PrivateKey::from_encoded_string(space_delim_strings[2])?;
         let consensus_public_key = Ed25519PublicKey::from_encoded_string(space_delim_strings[3])?;
-        let network_signing_key = Ed25519PublicKey::from_encoded_string(space_delim_strings[4])?;
-        let network_identity_key = x25519::PublicKey::from_encoded_string(space_delim_strings[5])?;
-        let network_address = NetworkAddress::from_str(space_delim_strings[6])?;
+        let network_identity_key = x25519::PublicKey::from_encoded_string(space_delim_strings[4])?;
+        let network_address = NetworkAddress::from_str(space_delim_strings[5])?;
         let network_address = RawNetworkAddress::try_from(&network_address)?;
-        let fullnode_identity_key = x25519::PublicKey::from_encoded_string(space_delim_strings[7])?;
-        let fullnode_network_address = NetworkAddress::from_str(space_delim_strings[8])?;
+        let fullnode_identity_key = x25519::PublicKey::from_encoded_string(space_delim_strings[6])?;
+        let fullnode_network_address = NetworkAddress::from_str(space_delim_strings[7])?;
```

Comment:
Is there a diagram somewhere that says what the order of these are?  Just for future reference could be helpful.

Reply:
Maybe @zekun000 knows, I don't know if anyone else is using this at this point in time and is unlikely to be useful come V1. At that point, management or something more in that domain is actually the tool to add / remove validators.